src/routers/reviewRouter.js

```diff
@@ -0,0 +1,12 @@
+const express = require("express");
+const reviewController  = require("../controllers/reviewController");
+const loginRequired  = require("../utils/auth");
+
+const router = express.Router();
+
+router.post("/add", loginRequired, reviewController.addReview);
+router.get("/get/:bookId", reviewController.getReview);
+router.put("/:id", loginRequired, reviewController.updateReview);
+router.delete("/delete/:bookId", loginRequired, reviewController.deleteReview);
```

Comment:
RESTful API에서는 엔드포인트에 동사를 적지 않습니다.
이미 메서드 자체에서 그 의미가 내포되어 있습니다!

해당 사항은 여러번 가이드 드렸던 사항으로 꼭! 꼭!!!! 잊지 말아주세요!!!!!!!!!
```suggestion
router.delete("/:bookId", loginRequired, reviewController.deleteReview);
```

Reply:
수정완료하였습니다!